src/resources/js/components/layout/SideBar.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+    <v-list
+        nav
+        dense
+    >
+        <v-list-item
+            v-for="(route, i) in routes"
+            :key="i"
+            :href="route.path"
+        >
+            <v-list-item-icon>
+                <v-icon>{{ route.icon }}</v-icon>
+            </v-list-item-icon>
+            <v-list-item-content>
+                <v-list-item-title v-text="route.name"></v-list-item-title>
+            </v-list-item-content>
+        </v-list-item>
+    </v-list>
+</template>
+
+<script>
+    import {router} from "../../router";
+
+    export default {
+        name: "SideBar",
+        data: () => ({
+            route: 0,
```

Comment:
I don't see (maybe I'm blind) where this ``route`` set to 0 is used in the file, could it be removed?

Reply:
it's right corrected on 9d654652147ff979958f027cdf7acbcf4688e8a1